src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -670,10 +670,6 @@ private async Task RestoreAsync(bool forceRestore, RestoreOperationSource restor
                     (_, __, ___) => CheckForMissingPackagesAsync(packages),
                     token);
             }
-
-            await _packageRestoreManager.RaisePackagesMissingEventForSolutionAsync(
```

Comment:
This is the old one right? This would probably regress the scenario where restore happens through the solution restore job and it owuldn't remove the yellow bar.

Reply:
Reverting this change